Language/Variables/Variable Scope & Qualifiers/volatile.adoc

```diff
@@ -77,16 +78,19 @@ void blink() {
 }
 ----
 
+Use an atomic block for atomic access to a 16-bit `int`:
 
 [source,arduino]
 ----
 #include <util/atomic.h> // this library includes the ATOMIC_BLOCK macro.
 volatile int input_from_interrupt;
 
+void loop() {
```

Comment:
I understand the motivation behind this change, but I don't like that it gives the impression that this is a complete sketch, when it is not. I also think that adding all the code to make it a complete sketch might end up burying the important part of the code. Perhaps the solution would be to add some comments that indicate where the rest of the code of a complete sketch would normally be?

Reply:
I agree, I was indeed considering this option too for the same reason. Will change it and add some comments!